content/handbook/product/product-processes/making-gifs.md

```
@@ -0,0 +1,149 @@
+
+## クイック＆イージー
+
+「クイック＆イージー」のプロセスは、パフォーマンスがそれほど重要でない日常用途向けです。GIF を作成する最も効率的な方法です。
+
+### ステップ 1
+
+必要なものだけを表示します。すべてを含める必要はありません。これは単一のバグ／機能のひと垣間です。記録領域を小さく、専用に保ってください。
+
+### ステップ 2
+
+すぐに GIF を出力する専用アプリを使用します。[ツール - オールインワン](#all-in-one) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)